aiida/sphinxext/__init__.py

```diff
@@ -11,10 +11,6 @@
 Defines reStructuredText directives to simplify documenting AiiDA and its plugins.
 """
 
-from __future__ import division
-from __future__ import print_function
-from __future__ import absolute_import
-
 from sphinx.errors import ExtensionError
 
 
```

Comment:
The try / except just below this is also a py2 workaround (for some reason I can't comment there :upside_down_face:)